protocols/569d34-part1/nuc_acid_purification_pt1.ot2.py

```diff
@@ -0,0 +1,134 @@
+from opentrons import labware, instruments, modules, robot
+from otcustomizers import StringSelection
+import math
+
+metadata = {
+    'protocolName': 'Nucleic Acid Purification: Part 1/2',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# create custom labware
+res_name = 'biomek_4_reservoir_40ml'
+if res_name not in labware.list():
+    labware.create(
+        res_name,
+        grid=(4, 1),
+        spacing=(7.4325, 0),
+        diameter=23.65,
+        depth=3.5,
```

Comment:
Just wanted to confirm this is the correct depth - seems too short to me

Reply:
good catch thanks